provision/provision-site.sh

```diff
@@ -484,6 +511,7 @@ function vvv_custom_folders() {
 }
 
 # -------------------------------
+vvv_set_php_cli_version()
```

Comment:
needs a doc block

Reply:
but no docblock, so it won't have auto-generated documentation when the docs site is updated

Reply:
See https://github.com/Varying-Vagrant-Vagrants/varyingvagrantvagrants.org/blob/main/inject-repo-documentation.sh and https://github.com/reconquest/shdoc

Reply:
and https://varyingvagrantvagrants.org/docs/en-US/function-reference/provision-provision-site/